src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,35 @@
+
+import { useEffect, useState } from "react";
+import { getArticleById } from "./api";
+import { useParams } from "react-router-dom";
+
+
+const SingleArticle = () => {
+    const [article, setArticle] = useState([])
+    const {article_id} = useParams();
+
+    useEffect(() => {
+        getArticleById(article_id).then(({data}) => {
+            setArticle(data.articles)
+        })
+
```

Comment:
Should add a loading pattern so the user isn't looking at a blank page while they wait for the article to load ✅ 
You can find an example of the loading pattern at the bottom of these notes https://notes.northcoders.com/courses/js-front-end/useEffect